packages/config-array/tests/config-array.test.js

```diff
@@ -1525,12 +1525,12 @@ describe("ConfigArray", () => {
 				);
 			});
 
-			it('should return "matched" when passed docx filename', () => {
+			it('should return "unconfigured" when passed docx filename', () => {
 				const filename = "sss.docx";
 
 				assert.strictEqual(
 					configs.getConfigStatus(filename),
-					"matched",
+					"unconfigured",
 				);
 			});
```

Comment:
This test was added in https://github.com/eslint/rewrite/pull/7, where part of the change was to update `isFileIgnored()` to return `true` only if the file is "really" ignored, that is, ignored by ignore patterns and not just unmatched. I _think_ the test was added to correlate with a similar test for `isFileIgnored()`,  and was rather describing the current behavior than insisting that it is correct in this case.

Reply:
Yes, in that test `getConfigStatus` is returning `"matched"` because one of the config objects contains a negated pattern, which is being treated as a non-universal match according to the current logic:

https://github.com/eslint/rewrite/blob/e84cbd7468765c25c01fc420b43d75a256533da2/packages/config-array/tests/config-array.test.js#L82-L87

This was the expected behavior prior to this fix.